api/notd/model.py

```diff
@@ -664,3 +664,9 @@ class MintedTokenCount:
     date: datetime.datetime
     mintedTokenCount: int
     newRegistryCount: int
+
+
+@dataclasses.dataclass
+class TradingHistory:
+    date: datetime.date
+    counts: Dict[str, int]
```

Comment:
might be easier to just inline the values here since you have them as zeros when not there, so we can just have date, buyCount, sellCount, mintCount as fields